src/Immediate.Apis.Analyzers/AnalyzerReleases.Shipped.md

```
@@ -46,3 +46,10 @@ IAPI0013 | ImmediateApis | Error | InvalidMapGroupAnalyzer
 Rule ID | Category | Severity | Notes
 --------|----------|----------|-------
 IAPI0010 | ImmediateApis | Error | InvalidRouteGroupNameAnalyzer
+
+## Release 6.3
+
+### New Rules
+Rule ID | Category | Severity | Notes
```

markdownlint-cli2 (0.23.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)